.circleci/config.yml

```diff
@@ -27,8 +35,52 @@ jobs:
               --publish-deployment << parameters.semgrep_deployment_id >> \
               --publish-token $SEMGREP_APP_TOKEN
 
+  pip-publish:
+    docker:
+      - image: cimg/python:3.8
+    environment:
+      TRIGGER_TAG: << pipeline.git.tag >>
+    steps:
+      - run:
+          name: clone main repo, publish tag that triggered the pipeline
+          command: |
+            mkdir -p ~/.ssh
+            ssh-keyscan -t rsa github.com >> ~/.ssh/known_hosts
+            git clone git@github.com:aflorithmic/apiaudio-python.git
+            cd apiaudio-python
+            git checkout $TRIGGER_TAG
+            pip3 install twine wheel
+            python3 setup.py sdist bdist_wheel
+            python3 -m twine upload --username $PIPY_USERNAME --password $PIPY_PASSWORD dist/*
```

Comment:
We should include slack notifications for fails/success. I will checkout from this branch and will add it for you. PR incoming